scripts/build-images

```diff
@@ -61,6 +61,13 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images-calico.txt
     ${REGISTRY}/rancher/mirrored-calico-apiserver:v3.26.1
 EOF
 
+xargs -n1 -t docker image pull --quiet << EOF > build/images-harvester-csi.txt
```

Comment:
These are already in `images-harvester.txt` down below; why are we adding another list that includes only the harvester CSI without the CPI?

Reply:
I added them because the test was failing without them.
But in fact the problem was that the images in `images-harvester.txt` were not using the `mirrored-` prefix so I corrected that instead.

Reply:
hey look, you're finding bugs already!